src/server.rs

```diff
@@ -259,11 +259,6 @@ pub(super) fn send_replication(
     server_tick: Res<ServerTick>,
     time: Res<Time>,
 ) -> Result<()> {
```

Comment:
If this system early outs with an error, the next time it runs (assuming there is no panic - currently there is an unwrap in the plugin but as far as this function is concerned that doesn't matter) the Updates/Mutations components will be in an invalid state.

Reply:
Hm... Maybe better undo the change then. I currently working on relations support and it turns out that I need to loop over mutation messages anyway :thinking:

Reply:
Yeah for now let's undo it, and then we can revisit for optimizations after relations is in.